ocrd_page_to_alto/convert.py

```diff
@@ -183,55 +184,60 @@ def convert_border(self):
         page_height = self.page_page.imageHeight
         setxml(self.alto_page, 'WIDTH', page_width)
         setxml(self.alto_page, 'HEIGHT',  page_height)
-        page_printspace = self.page_page.get_PrintSpace()
-        dummy_printspace = False
-        if page_printspace is None:
-            self.logger.warning("PAGE-XML has no PrintSpace, trying to fall back to Border")
-            page_printspace = self.page_page.get_Border()
-            if page_printspace is None:
-                dummy_printspace = True
+        page_border = self.page_page.get_Border()
+        page_pspace = self.page_page.get_PrintSpace()
+        if page_pspace is None and not page_border is None:
```

Comment:
Small thing but I prefer `page_border is not None` over `not page_border is None` for readability.

Reply:
Done.